tools/github/client.go

```diff
@@ -110,57 +108,22 @@ func CacheFilePath() (string, error) {
 }
 
 var (
-	// Correctly formatted FerretDB issue.
-	// It returns repository name and the issue number as it's submatches.
-	urlRE = regexp.MustCompile(`^\Qhttps://github.com/FerretDB/\E([-\w]+)/issues/(\d+)$`)
-
 	// ErrIncorrectURL indicates that FerretDB issue URL is formatted incorrectly.
 	ErrIncorrectURL = errors.New("invalid TODO: incorrect format")
 
 	// ErrIncorrectIssueNumber indicates that FerretDB issue number is formatted incorrectly.
 	ErrIncorrectIssueNumber = errors.New("invalid TODO: incorrect issue number")
 )
 
-// parseIssueURL takes the properly formatted FerretDB issue URL and returns it's
-// repository name and issue number.
-// If the issue number or URL formatting is incorrect, the error is returned.
-func parseIssueURL(line string) (repo string, num int, err error) {
-	match := urlRE.FindStringSubmatch(line)
-
-	if len(match) != 3 {
-		err = ErrIncorrectURL
-		return
-	}
-
-	repo = match[1]
-
-	num, err = strconv.Atoi(match[2])
-
-	switch {
-	case err != nil:
-		panic(err)
-	case num <= 0:
-		err = ErrIncorrectIssueNumber
-		return
-	default:
-		return
-	}
-}
-
 // IssueStatus returns issue status.
 // It uses cache.
 //
 // If URL formatting is incorrect it returns `ErrIncorrectURL`, or `ErrIncorrectIssueNumber` error.
```

Comment:
Is it still?

That's something I noticed in the genwrap PR, too – only small parts are updated, making other things incorrect. If something is being changed, both callers and comments should be analyzed and updated.

Reply:
Thanks, the comment was no longer correct. Updated relevant parts :pray: